llm/llama-4-finetuning/README.md

```diff
@@ -7,32 +7,40 @@
 
 Meta's Llama 4 represents the next generation of open-source large language models, featuring advanced capabilities with the **Llama-4-Maverick-17B-128E model** - a 400B  parameter (17B active) Mixture of Experts (MoE) architecture with 128 experts.
 
-This guide shows how to use [SkyPilot](https://github.com/skypilot-org/skypilot) and [torchtune](https://pytorch.org/torchtune/stable/index.html) to **finetune Llama 4 on your own infra**. Everything is packaged in a simple [SkyPilot YAML](https://docs.skypilot.co/en/latest/getting-started/quickstart.html), that can be launched with one command on your infra: 
+This guide shows how to use [SkyPilot](https://github.com/skypilot-org/skypilot) and various frameworks to **finetune Llama 4 on your own infra**. Everything is packaged in simple [SkyPilot YAMLs](https://docs.skypilot.co/en/latest/getting-started/quickstart.html), that can be launched with one command on your infra: 
 - Kubernetes cluster
 - Cloud accounts ([16+ clouds supported](https://docs.skypilot.co/en/latest/getting-started/installation.html))
 
-## Finetune Llama 4 with SkyPilot
-We will use [torchtune](https://pytorch.org/torchtune/stable/index.html) to finetune Llama 4 Maverick.
+## 📁 Available Configuration Files
 
-This model requires at least 2 nodes with 8x H200 GPUs each.
+Choose the right configuration for your needs:
 
-To set up the environment for launching the finetuning job, finish the [Appendix: Preparation](#appendix-preparation) section first.
+| **Configuration File** | **Requirements** | **Description** |
+|------------------------|------------------|-----------------|
+| 🌟 **llama-4-maverick-sft.yaml** | **4 nodes**<br>16x H200 GPUs<br>1000+ GB CPU memory | **✅ RECOMMENDED** - CPU-friendly full finetuning using LLaMA-Factory with CPU offloading. Best starting point for most users. |
```

Comment:
This doesn't use `LLaMA-Factory` anymore, right? Also maybe rephrase to `Full finetuning using torchtune with CPU offloading. Best starting point for most users.`